app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/label.tsx

```diff
@@ -0,0 +1,32 @@
+/** @file A label that may have a warning symbol */
+import * as react from "react";
+
+export enum Status {
+	none = "none",
+	warning = "warning",
+	severeWarning = "severe-warning",
```

Comment:
strange indentation - did you seta's instead of spaces? Please be sure that everywhere you are using spaces and the indentation is consistent.